src/server/.tests/passport/strategyUtils.spec.js

```diff
@@ -0,0 +1,168 @@
+		})
+		describe('#serializeHandler', () => {
+			it('should call next when no error is given', () => {
+				//given
+				const error = null
+
+				//when
+				suite.strategyHandlersMock.serializeHandler(error)
+
+				//then
+				assert.isTrue(suite.nextSpy.calledOnce)
+			})
+
+			it('should respond with error status code and description', () => {
+				//given
+				const error = {
+					code: 401,
+					description: 'UNAUTHORIZED'
+				}
+
+				//when
+				suite.strategyHandlersMock.serializeHandler(error)
+
+				//then
+				const statusCode = suite.responseMock._getStatusCode()
+				const payload = suite.responseMock._getData()
+				const expectedStatusCode = error.code
+				const expectedPayload = JSON.stringify({
+					description: error.description
+				})
+
+				assert.strictEqual(statusCode, expectedStatusCode)
+				assert.strictEqual(payload, expectedPayload)
+			})
+		})
+
+		describe('#loginStrategyHandler', () => {
+			it('should respond with error status code and description, when error is given', () => {
+				//given
+				const error = {
+					code: 401,
+					description: 'UNAUTHORIZED'
+				}
+				const userInstance = {}
+				const info = null
+
+				//when
+				suite.strategyHandlersMock.loginStrategyHandler(error, userInstance, info)
+
+				//then
+				const statusCode = suite.responseMock._getStatusCode()
+				const payload = suite.responseMock._getData()
+				const expectedStatusCode = error.code
+				const expectedPayload = JSON.stringify({
+					description: error.description
+				})
+
+				assert.strictEqual(statusCode, expectedStatusCode)
+				assert.strictEqual(payload, expectedPayload)
+
+			})
+
+			it('should respond with {description: UNATHORIZED , code: 401}, when userInstance is null', () => {
+				//given
+				const error = null
+				const userInstance = null
+				const info = null
+
+				//when
+				suite.strategyHandlersMock.loginStrategyHandler(error, userInstance, info)
+
+				//then
+				const statusCode = suite.responseMock._getStatusCode()
+				const payload = suite.responseMock._getData()
+
+				const expectedError =  responseManager.MESSAGES.errors.UNAUTHORIZED
+				const expectedStatusCode = expectedError.code
+				const expectedPayload = JSON.stringify({
+					description: expectedError.description
+				})
+
+				assert.strictEqual(statusCode, expectedStatusCode)
+				assert.strictEqual(payload, expectedPayload)
+
+			})
+
+			it('should call req.logIn with arguments userInstance and serializeHandler, when was passed correct userInstance without any error', () => {
+				//given
+				const error = null
+				const userInstance = {
+					username: 'Rick',
+					password: 'TheSmartestPersonInMultiUniverse'
+				}
+				const info = null
+
+				//when
+				suite.strategyHandlersMock.loginStrategyHandler(error, userInstance, info)
+
+				//then
+				const logInSpy = suite.requestMock.logIn
+				assert.isTrue(logInSpy.calledOnce)
```

Comment:
Test description states, that you check what arguments is `req.logIn` called with, but you just check if it was called once.

Reply:
ok